### DISCUSSION TITLE
For the color picker squares how to make them radio buttons with a selected highlight around?

### DISCUSSION THREAD

**Discussion by spacerobotTR** [themesberg/flowbite#365] [Q&A]
![image](https://user-images.githubusercontent.com/3800436/208282981-6498e88b-e423-45bc-a7b0-d784c146fe5e.png)

How to make this group of buttons radio buttons and have them highlight when one is selected?

**Comment by zoltanszogyenyi** [comment]
Hello @spacerobotTR,

I would use pseudo selectors and hide the default checkbox / radio styles.
